consumer/sm_context.go

```diff
@@ -519,7 +563,7 @@ func SendUpdateSmContextRequest(smContext *amf_context.SmContext,
 func SendReleaseSmContextRequest(ue *amf_context.AmfUe, smContext *amf_context.SmContext,
 	cause *amf_context.CauseAll, n2SmInfoType models.N2SmInfoType,
 	n2Info []byte,
-) (detail *models.ProblemDetails, err error) {
+) (*models.ProblemDetails, error) {
```

Comment:
I do not think this change is needed. That is, revert this change to
```go
func SendReleaseSmContextRequest(ue *amf_context.AmfUe, smContext *amf_context.SmContext,
	cause *amf_context.CauseAll, n2SmInfoType models.N2SmInfoType,
	n2Info []byte,
) (detail *models.ProblemDetails, err error) {
```
and remove their definitions from below
```go
	var detail *models.ProblemDetails
	var err error

```